package.json

```diff
@@ -37,6 +38,7 @@
   ],
   "devDependencies": {
     "ava": "*",
-    "xo": "*"
+    "xo": "*",
+    "matcha":"^0.7.0"
```

Comment:
These should be listed in alphabetical order. You could run `npm install --save-dev matcha` and `npm` will put them in the correct order automatically.